.github/workflows/integrationTest.yml

```diff
@@ -10,6 +10,10 @@ on:
       - master
   workflow_dispatch:
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref_name }}
+  cancel-in-progress: true
```

Comment:
I don't have a lot of insight on the integration tests. If we cancel it in progress, will the test still tear down the resources created during the integration test?

Reply:
Yes since the integration test use [always as a condition](https://github.com/aws/amazon-cloudwatch-agent/blob/master/.github/workflows/integrationTest.yml#L62), it also covers the ```canceled``` condition too. Therefore, it would cancel the progress and run to the ```terraform destroy``` command before starting another workflow